.github/workflows/ci.yml

```diff
@@ -38,22 +47,17 @@ jobs:
         uses: actions/cache@v3
         with:
           path: packages/*/dist
-          key: ${{ matrix.os }}-node-v${{ matrix.node }}-${{ github.sha }}
+          key: ubuntu-latest-node-v18-${{ github.sha }}
 
   lint:
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-        node: [16]
+    runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v4
       - run: corepack enable
       - uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node }}
+          node-version: 20
```

Comment:
Is there a reason why lint runs on `20` while other steps run on `18`? If so, may be worth commenting why, otherwise it looks like a typo

Reply:
there's no particular reason, other than this is the current node version and this is a lint step so it's not important it runs on an older version to ensure compatibility